async.go

```diff
@@ -215,7 +215,7 @@ func (sr *snowflakeRestful) getAsyncOrStatusWithPanic(
 		if resp != nil {
 			statusCode = fmt.Sprint(resp.StatusCode)
 		}
-		panicMessgae := fmt.Sprintf("Deadline set: %t, deadline value: %v, startTime: %v, timeout value: %v, statusCode: %s, stackTrace: ", ok, deadline, startTime, timeout, statusCode, debug.Stack())
+		panicMessgae := fmt.Sprintf("Deadline set: %t, deadline value: %v, startTime: %v, timeout value: %v, statusCode: %s, stackTrace: %s", ok, deadline, startTime, timeout, statusCode, string(debug.Stack()))
```

Comment:
so messy :). why not make a proper struct like:
```
  panicMessage := struct {
     deadline time.Duration,
    ..
    stack: []uintptr,
  } { 
     deadline: deadline,
    .... 
  }

  panicMesasge.stack = runtime.Callers(1, panicMessage.stack) 
``` 
 and when capturing on the other side you can make a proper error out of it by wrapping in something that implements Error() and maybe wrap it in `pkg/errors/errorWithStack`.